test/unittests/statetest_loader_test.cpp

```diff
@@ -0,0 +1,40 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <gtest/gtest.h>
+#include <test/statetest/statetest.hpp>
+
+using namespace evmone;
+
+TEST(statetest_loader, load_empty_test)
+{
+    std::istringstream s{"{}"};
+    EXPECT_THROW(test::load_state_test(s), std::invalid_argument);
+}
+
+TEST(statetest_loader, load_minimal_test)
+{
+    std::istringstream s{R"({
+        "test": {
+            "_info": {},
+            "pre": {},
+            "transaction": {
+                "gasPrice": "",
```

Comment:
Many value parsing is overly permissive. I think it is fine to leave it for now like that.